lib/liftoff/xcodeproj_helper.rb

```diff
@@ -53,18 +53,18 @@ def add_script_phases(scripts)
       end
     end
 
-    def perform_extra_config(extra_config)
-      if extra_config
-        extra_config.each do |name, settings|
-          if name.downcase == "all"
-            object = target
-          else
-            object = target.build_settings(name)
-          end
+    def perform_extra_config(app_config, test_config)
+      {app_config => target, test_config => test_target}.each do |config, t|
```

Comment:
What about renaming `t` to `target`, and renaming the `target` method below to something a little more descriptive?